snoop/digest.py

```diff
@@ -62,8 +62,9 @@ def digest(doc):
 
     if emails.is_email(doc):
         data.update(emails.parse_email(doc))
-        if doc.flags.get('pgp'):
-            data['pgp'] = doc.flags.get('pgp')
+
+    if doc.flags.get('pgp'):
```

Comment:
How about `if 'pgp' in doc.flags`? Then we'd just propagate the flag if present, without caring about its possible values.